watcher.js

```diff
@@ -0,0 +1,25 @@
+const chokidar  = require('chokidar')
```

Comment:
can you please move this file into a `scripts` folder where we can further keep all setups and stuff like that?

Reply:
Sure 👍